tests/test_suspend_resume/LICENSE.rst

```diff
@@ -0,0 +1,84 @@
+*******************************
```

Comment:
On 2021-05-05, @lucianomartin removed the license files from the test directories as part of the preparation for making the repo public.  You can see this change in commit c8a7bd8.  I asked him to do so because the tests are not distributed separately from the repo, so there's no need for an additional license file and having duplicates increases the risk of them diverging.  Please do not reintroduce per-test license files.

Reply:
Copy pasta. Agree'd should not be duplicated.